README.md

```diff
@@ -118,6 +118,65 @@ Whilst the `asset-pipline` script is custom, it leverages a lot of existing libs
 
 For the `asset-pipeline` script to do its thing, all you need to do is refer to all assets with a path beginning with `/blog/assets/`. If you do that, everything else is handled for you ✨
 
+## Deployment strategy
+
+Currently, we use a "simple" deployment strategy based on git tags and Circle CI to deploy this blog. We have three different tag patterns:
+
+- `<year>.<month>.<date>-stage` (e.g., `2021.07.22-stage`): these tags will be deployed to our [-stage instance][stage]
+- `<year>.<month>.<date>` (e.g., `2021.07.22`): these tags will be deployed to our [-prod instance][prod]
+- `x.y.z` (e.g., `1.3.0`): this is the `version` in the `package.json` and it is used to version the WordPress theme that can be constructed in this project
+
+Each deployment will fetch the content from the (headless) WordPress instance, which ensures that the most recent content will be deployed.
+
+### -dev
+
+All commits on the `main` branch are automatically deployed to our [-dev instance][dev]. The content of the blog might not always be up-to-date.
+
+#### How to redeploy -dev?
+
+Either push a new commit to the `main` branch or go to [the Circle CI page][circle-addons-blog], select the `main` branch and re-run the most recent `default-workflow` workflow.
+
+### -stage
+
+As mentioned previously, git tags matching `<year>.<month>.<date>-stage` will be automatically deployed to our [-stage instance][stage].
+
+In addition, we automatically update -stage every 3 hours using a CRON task configured in Circle CI (see `autodeploy-stage` workflow in the [Circle CI configuration](.circleci/config.yml)). This task looks for the most recent git tag matching the pattern above and deploy it. As mentioned previously, the content is fetched from the WordPress API every time.
+
+#### How to redeploy -stage?
+
+Either wait up to 3 hours or go to [the Circle CI page][circle-addons-blog] and re-run the `autodeploy-stage` workflow (or the mot recent "stage" tag).
+
+### -prod
+
+**Important:** when deploying to production, please deploy to -stage first.
+
+Git tags matching `<year>.<month>.<date>` will be automatically deployed to our [-prod instance][prod].
+
+A git tag for production should point to a commit already tagged for stage so that we can deploy the exact same commit in both environments:
```

Comment:
Thanks for including this, it's an important detail and it's easy to miss and accidentally include an extra commit.